src/main/java/Todo.java

```diff
@@ -0,0 +1,10 @@
+public class Todo extends Task{
+    public Todo(String label) {
+        super(label);
+    }
+
+    @Override
+    public String getTypeString() {
+        return "[T]";
+    }
+}
```

Comment:
Generally the absence of doubly nested code suggests that you are aware that the expected execution path should be made [obvious](https://nus-cs2113-ay2021s2.github.io/website/se-book-adapted/chapters/codeQuality.html#make-the-happy-path-prominent). Good work! 👍